src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  let totalAmount;
+
+  if (days >= 7) {
+    totalAmount = (days * costPerDay) - 50;
```

Comment:
You can save `days * costPerDay` to a variable too. 
Also, would be great to create variables for all important values (7, 50, 3, 20 etc.) to make your cleaner and more understandable